baiji/connection.py

```diff
@@ -66,7 +66,7 @@ def _lookup(self, bucket_name, key, cache_buckets=None):
 
         return bucket.lookup(key)
 
-    def cp(self, key_or_file_from, key_or_file_to, force=False, progress=False, policy=None, preserve_acl=False, encoding=None, encrypt=True, gzip=False, content_type=None, guess_content_type=False, metadata=None, skip=False, validate=True, max_size=None):
+    def cp(self, key_or_file_from, key_or_file_to, force=False, progress=False, policy=None, preserve_acl=False, encoding=None, encrypt=True, gzip=False, content_type=None, guess_content_type=False, metadata=None, skip=False, validate=True, max_size=None, version=None):
```

Comment:
You're using both `version` and `version_id` throughout... Is there a reason why you chose `version` in some places and `version_id` in others?